src_modules/player/index.ts

```diff
@@ -1,6 +1,3 @@
-// extcord module
-// requires ffmpeg-static @distube/ytdl-core ytsr ytpl
-
```

Comment:
Revert the removal of the comments. They are needed for the module to work properly.